superbench/benchmarks/micro_benchmarks/gpu_copy_performance/gpu_copy.cu

```diff
@@ -313,6 +313,23 @@ int SetGpu(int gpu_id) {
     return 0;
 }
 
+// Check if its NUMA node has CPUs.
+bool HasCPUsForNumaNode(int node) {
+    struct bitmask *bm = numa_allocate_nodemask();
+
+    int numa_err = numa_node_to_cpus(node, bm);
+    if (numa_err != 0) {
+        fprintf(stderr, "numa_node_to_cpus error on node: %d, error code: %d\n", node, numa_err);
```

Comment:
Sorry but by I meant errno which is set by numa_node_to_cpus (like ERANGE mentioned in https://linux.die.net/man/3/numa_node_to_cpus) instead of the return code of numa_node_to_cpus.